eclair-core/src/main/scala/fr/acinq/eclair/Setup.scala

```diff
@@ -124,6 +124,7 @@ class Setup(val datadir: File,
     // @formatter:off
     override def getFeeratePerKb(target: Int): FeeratePerKB = feeratesPerKB.get().feePerBlock(target)
     override def getFeeratePerKw(target: Int): FeeratePerKw = feeratesPerKw.get().feePerBlock(target)
+    override def mempoolMinFee(): FeeratePerKw = feeratesPerKw.get().mempoolMinFee
```

Comment:
For consistency? Not pretty I know:

```suggestion
    override def getMempoolMinFeeratePerKw(): FeeratePerKw = feeratesPerKw.get().mempoolMinFee
```

Reply:
Consistency is good, done in https://github.com/ACINQ/eclair/pull/1980/commits/9574bb183cab6dcc7f4ba2e458286f7d670a50db